spec/features/user_can_submit_posts_spec.rb

```diff
@@ -1,11 +1,11 @@
 require 'rails_helper'
 
-RSpec.feature "Timeline", type: :feature do
-  scenario "Can submit posts and view them" do
-    visit "/posts"
-    click_link "New post"
-    fill_in "Message", with: "Hello, world!"
-    click_button "Submit"
-    expect(page).to have_content("Hello, world!")
-  end
-end
+# RSpec.feature "Timeline", type: :feature do
```

Comment:
We've commented this out for now because we need extra functionality: to allow our user to sign up/log in before being able to post... Can we make our tests pass by adding a default user to our user table (at the beginning of the tests) and providing this user id when making a post?

Reply:
I can try. Mal mentioned something quite interesting about creating 'fake users' to run the tests.